src/main/g8/src/main/scala/$package$/AkkaQuickstart.scala

```diff
@@ -50,7 +50,7 @@ class Printer extends Actor with ActorLogging {
 
   def receive = {
     case Greeting(greeting) =>
-      log.info(s"Greeting received (from ${sender()}): $greeting")
+      log.info("Greeting received (from " + sender() + "): " + greeting)
```

Comment:
This is a bit of a shame, wouldn't there be some other way to get rid of the `Fix `mismatched input '$' expecting LPAREN` warning?

Reply:
I know we can use backslash `\` to handle this, 
```scala
log.info(s"Greeting received (from \${sender()}): \$greeting")
```
but paradox didn't handle this. It will appear those escape characters at Lightbend Tech Hub. That's why I chose this way. If there is another way to handle this, please let me know. Thanks! :)

Reply:
Ah, so it's an interaction between paradox and g8: since we removed `verbatim = *.scala` (to make the variable substitution for the package name work) it now also tries to interpret the `$` signs in the rest of the code, but paradox isn't prepared for that. This is fine for now then.